### PR TITLE
mark up Blog page is down

### PR DESCRIPTION
**Pull request by sidvlad**

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+                Discord
+            </a>
+            <a href="#" class="btn btn--icon btn--color">
+                <img src="images/connect.svg" width="24" height="24" alt="connect">
+                Connect
+            </a>
+        </div>
+    </header>
+    <section class="blog-intro">
+        <h2 class="blog-intro__title">Our blog</h2>
```
Comment (andacenko) [inline blog.html]: This is `<h1>`

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+        <ul class="blog-intro__nav">
+            <li href="#" class="blog-intro__item">Home</li>
+            <li href="#" class="blog-intro__item">Our blog</li>
+            <li href="#" class="blog-intro__item">The Basilisks NFT</li>
+        </ul>
```
Comment (andacenko) [inline blog.html:54]: This is typical breadcrumbs
ul.breadcrumbs > li > a > text
But last item! ul.breadcrumbs > li> text

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+    </section>
+     <main class="blog-wrapper">
```
Comment (andacenko) [inline blog.html]: class="main main--two-col"
It's can be useful not for blog page only

File: blog.html [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+        <aside class="blog-sidebar">
```
Comment (andacenko) [inline blog.html]: class="sidebar"

File: blog.html [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+            <div class="blog-sidebar__block">
+                <h3 class="blog-sidebar__title">Categories</h3>
```
Comment (andacenko) [inline blog.html]: `<h2>` or `<strong>` keep the order of headings

File: blog.html [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+                    <li class="categories__item"><a href="#">Metaverse <span>(24)</span></a></li>
+                    <li class="categories__item"><a href="#">Gaming <span>(24)</span></a></li>
+                    <li class="categories__item"><a href="#">Art <span>(24)</span></a></li>
+                    <li class="categories__item"><a href="#">Robot <span>(24)</span></a></li>
+                    <li class="categories__item"><a href="#">Launchpad <span>(24)</span></a></li>
+                    <li class="categories__item"><a href="#">Crypto <span>(430)</span></a></li>
+                </ul>
+            </div>
+            <div class="blog-sidebar__block">
+                <h3 class="blog-sidebar__title">Tag</h3>
```
Comment (andacenko) [inline blog.html]: The same, as above

File: blog.html [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+                    <li class="tags__item"><a href="#">NFT</a></li>
+                    <li class="tags__item"><a href="#">ICO</a></li>
+                    <li class="tags__item"><a href="#">Robot</a></li>
+                    <li class="tags__item"><a href="#">Crypto</a></li>
+                    <li class="tags__item"><a href="#">Launchpad</a></li>
+                    <li class="tags__item"><a href="#">Metaverse</a></li>
+                </ul>
+            </div>
+        </aside>
+        <div class="blog-content">
```
Comment (andacenko) [inline blog.html]: This whole block it's `<article>`

File: blog.html [hunk 8]
```diff
@@ -0,0 +1,251 @@
...
+                    <ul class="blog__list">
+                        <li>The Basilisks that players collect are represented with NFTs on Ethereum’s blockchain with real-world value. Players use the Basilisks they
+                             have gathered to battle other players to win ether (ETH).
+                        </li>
+                        <li>
+                            In Balthazar Dragons, players are immersed in a 3D open world to explore and capture dragon-like beasts called Basilisks.  Balthazar Dragons is 
+                            an upcoming fantasy role-playing game developed on the Ethereum blockchain by a decentralized autonomous organization (DAO) called the Balthazar DAO.
+                        </li>
+                    </ul>
+                    <ol class="blog__list blog__list--mark">
```
Comment (andacenko) [inline blog.html]: There is `<ul>` with class

File: blog.html [hunk 9]
```diff
@@ -0,0 +1,251 @@
...
+                <figcaption class="blog-content__caption">Photo caption</figcaption>
+            </figure>
+            <article class="blog-content__info">
+                <h2 class="subtitle">The Basilisks NFT</h2>
+                    <a href="#" class="btn btn__img btn__color">Gaming</a>
+                    <div class="blog__info">
+                        <p class="blog__name">Tony Nguyen</p>
+                        <time datetime="2022-06-20" class="blog__date">20 Jun 2022</time>
+                    </div>
+                    <p class="blog__text">The Basilisks that <mark>players</mark> collect are represented with NFTs on Ethereum’s blockchain with real-world value. Players use the
```
Comment (andacenko) [inline blog.html]: No need classes on the `<p> <a> <ul>` (except ul.blog__list--mark with blue bullets) - it's  would be 
WYSIWYG content

File: blog.html [hunk 10]
```diff
@@ -0,0 +1,251 @@
...
+                        <li>List name</li>
+                        <li>List name</li>
+                        <li>List name</li>
+                    </ol>
+                    <div class="blog__blockquote">
+                        <blockquote cite="https://">
+                            <p class="blog__blockquote--text">The first huge metaverse arena to arrange live sports, hang out and even make bets and wagers</p>
+                        </blockquote>
+                        <p class="blog__blockquote--rating">8.5/ 10<cite>Markout Corporation</cite></p>
+                    </div>
```
Comment (andacenko) [inline blog.html]: Great job!

File: blog.html [hunk 11]
```diff
@@ -0,0 +1,251 @@
...
+                    <li class="tags__item"><a href="#">Metaverse</a></li>
+                </ul>
+            </div>
+        </aside>
+        <div class="blog-content">
+            <figure class="blog-content__photo">
+                <img src="images/blog-photo.jpg" width="920" height="557" alt="dragon with man">
+                <figcaption class="blog-content__caption">Photo caption</figcaption>
+            </figure>
+            <article class="blog-content__info">
```
Comment (andacenko) [inline blog.html]: Just `<div>`

File: blog.html [hunk 12]
```diff
@@ -0,0 +1,251 @@
...
+            <figure class="blog__photo">
+                <img src="images/blog-photo.jpg" alt="Basilisk NFT">
+                <figcaption class="blog__photo-caption">Photo caption</figcaption>
+            </figure>
+
+            <figure class="blog__photo">
+                <img src="images/blog-photo.jpg" alt="Basilisk NFT">
+                <figcaption class="blog__photo-caption">Photo caption</figcaption>
+            </figure>
+
```
Comment (andacenko) [inline blog.html]: Please, wrap two `<figure>` in the `<div>`

File: blog.html [hunk 13]
```diff
@@ -0,0 +1,251 @@
...
+            <article class="blog__info">
```
Comment (andacenko) [inline blog.html]: `<div>`

File: blog.html [hunk 14]
```diff
@@ -0,0 +1,251 @@
...
+                <h3 class="intro__subtitle">NFTs are digital assets that represent real-world objects</h3>
+                <p class="blog__text">Setting up your avatar and personalizing your nickname on Binance is quick and easy. Users can go to binance.com under
```
Comment (andacenko) [inline blog.html]: delete class, just `<p>`

File: blog.html [hunk 15]
```diff
@@ -0,0 +1,251 @@
...
+                    <a href="#" class="blog-article__tag">NFT</a>
+                    <a href="#" class="blog-article__tag">ICO</a>
+                    <a href="#" class="blog-article__tag">Crypto</a>
+                </div>
+
+                <div class="blog__share">
+                    <span class="blog-article__share-title">Share:</span>
+                    <a href="#" class="blog-article__share-link">Twitter</a>
+                    <a href="#" class="blog-article__share-link">Facebook</a>
+                    <a href="#" class="blog-article__share-link">LinkedIn</a>
```
Comment (andacenko) [inline blog.html]: It's ul > li > a > img 
Class for ul could be .social-networks.social-networks--alt   or share-social-networks

File: blog.html [hunk 16]
```diff
@@ -0,0 +1,251 @@
...
+                    your new unique nickname, it will be standardized and used across the Binance ecosystem.</p>
+
+                <h3 class="intro__subtitle">NFT Avatar Generators</h3>
+                <p>You can find a range of NFT marketplaces where you can buy existing NFTs or list your own NFTs for sale.</p>
+
+                <div class="blog__tags">
+                    <span class="blog-article__tags-title">Tags:</span>
+                    <a href="#" class="blog-article__tag">NFT</a>
+                    <a href="#" class="blog-article__tag">ICO</a>
+                    <a href="#" class="blog-article__tag">Crypto</a>
```
Comment (andacenko) [inline blog.html]: It's ul > li > a > text

File: blog.html [hunk 17]
```diff
@@ -0,0 +1,251 @@
...
+                </div>
+
+                <div class="blog__share">
+                    <span class="blog-article__share-title">Share:</span>
+                    <a href="#" class="blog-article__share-link">Twitter</a>
+                    <a href="#" class="blog-article__share-link">Facebook</a>
+                    <a href="#" class="blog-article__share-link">LinkedIn</a>
+                </div>
```
Comment (andacenko) [inline blog.html:170]: blog__tags and blog__share wrap into `<footer class="">`
It's footer of article

File: blog.html (hunk 1 above)
Comment (sidvlad) [reply blog.html]: done

File: blog.html (hunk 2 above)
Comment (sidvlad) [reply blog.html:54]: done

File: blog.html (hunk 3 above)
Comment (sidvlad) [reply blog.html]: done

File: blog.html (hunk 4 above)
Comment (sidvlad) [reply blog.html]: done

File: blog.html (hunk 5 above)
Comment (sidvlad) [reply blog.html]: done. use strong tag

File: blog.html (hunk 17 above)
Comment (sidvlad) [reply blog.html:170]: done

File: blog.html (hunk 16 above)
Comment (sidvlad) [reply blog.html]: done

File: blog.html (hunk 15 above)
Comment (sidvlad) [reply blog.html]: done

File: blog.html (hunk 14 above)
Comment (sidvlad) [reply blog.html]: done

File: blog.html (hunk 7 above)
Comment (andacenko) [reply blog.html]: Change div.blog-content to `<article>`

File: blog.html [hunk 18]
```diff
@@ -0,0 +1,262 @@
...
+                <p class="blog__name">Tony Nguyen</p>
+                <time datetime="2022-06-20" class="blog__date">20 Jun 2022</time>
+            </div>
+            <p>The Basilisks that <mark>players</mark> collect are represented with NFTs on Ethereum’s blockchain with real-world value. Players use the 
+                <a href="#" class="blog__text--link">Basilisks they have</a> gathered to battle other players to win ether (ETH).
+                In Balthazar Dragons, players are immersed in a 3D open world to explore and capture dragon-like beasts called Basilisks.  
+                Balthazar Dragons is an upcoming fantasy role-playing game developed on the Ethereum blockchain by a decentralized autonomous 
+                organization (DAO) called the Balthazar DAO.
+            </p>
+            <ul class="blog__list">
```
Comment (andacenko) [inline blog.html]: delete class

File: blog.html [hunk 19]
```diff
@@ -0,0 +1,262 @@
...
+
+            <figure class="blog__photo">
+                <img src="images/blog-photo.jpg" alt="Basilisk NFT">
+                <figcaption class="blog__photo-caption">Photo caption</figcaption>
+            </figure>
+
+            <figure class="blog__photo">
+                <img src="images/blog-photo.jpg" alt="Basilisk NFT">
+                <figcaption class="blog__photo-caption">Photo caption</figcaption>
+            </figure>
```
Comment (andacenko) [inline blog.html]: Please, wrap two `<figure>` in the `<div>`

File: blog.html (hunk 18 above)
Comment (sidvlad) [reply blog.html]: deleted, done

File: blog.html (hunk 19 above)
Comment (sidvlad) [reply blog.html]: done

File: blog.html (hunk 13 above)
Comment (sidvlad) [reply blog.html]: done

**Comment by sidvlad** [comment]
fixed

**Review by andacenko — APPROVED** [review]

**Pull request merged**